jest.config.ts

```diff
@@ -0,0 +1,31 @@
+const esmModules = [
+  '((jest-)?react-native|@react-native(-community)?)',
+  'expo(nent)?',
+  '@expo(nent)/.*?',
+  '@expo-google-fonts/.*',
+  'react-navigation',
+  '@react-navigation/.*',
+  '@unimodules/.*',
+  'unimodules',
+  'sentry-expo',
+  'native-base',
+  'react-native-svg',
+].join('|');
+
+export default {
```

Comment:
I'm a little hesitant about switching to SWC since the current babel based setup works quite well and the time it takes to execute all the tests is only like 5 seconds so it doesn't take that long any way 🤔

Reply:
@Temzasse 
Of course, I get lit of it,  if you don't like it.
But, let me explain it to you  a little, the configuration is very simple and easy to replace. All you need to do is `@swc/jest` to `babel-jest`, and it marks 5 times faster than babel-jest in my project's test .  In my local in this project, it passed test less than 1 sec.
The case that SWC is fragile is when you compress and mangling codes and with using decorators, and vercel is already using it in their libary, Next.js so in almost case you don't need to worry about it if you don't use conditions described above.

But, you don't feel pain to follow up this tool, I'll remove it.